### ISSUE TITLE
Where does it output images too?

### ISSUE THREAD

**Issue by Rezboa** [LordOfPolls/Imagefap-Downloader#2]
It goes so fast it doesn't show or tell me where the images are outputting too, even in the readme.

Where do they go?

**Comment by logan120** [comment]
Good question, i'm not sure either. It seems to ask for the link then process for a couple of seconds then closes the window. Seeing it in slow-motion it there doesn't seem to be anything after the "Processing" line. If you search for all pictures in your C-Drive no pictures fro that gallery seem to show up either. I would expect the creater of this program or another user to figure out if the program just doesn't work.

**Comment by logan120** [comment]
Ok figured it out. So looking at the code when instead of putting in the full link your supposed too only put in the galleryId. This caused by lines  " url = "http://www.imagefap.com/pictures/"  " and " 
 galleryId = URL.replace(url,"")  ". So for example instead of putting "https://www.imagefap.com/pictures/8695527/mixed-bag" you would put "8695527".

**Comment by LordOfPolls** [comment]
Hello there, 

Yeah i made this script quite a while ago, and stopped paying attention to it. Both it and the rule34 downloader are currently broken. As i have a lot of time on my hands ill get them working again. 

As for where this script saves to, it saves to the same directory as you run it in. But as i say, its broken right now. Once i get it running again, ill implement an option to choose save location, like r34 downloader does.

**Comment by LordOfPolls** [comment]
Found the issue, 

At some point imagefap moved from ``http`` to ``https`` and restructured their url format, evidently when i made this 3 years ago, i was very naive in how i grabbed the gallery ID from the url. I've updated accordingly. The script *should* now work fine. 

I will push a few more commits to add functionality and refactor the code over the next few days. But as of this comment, its working again
